src/argo-bot.ts

```diff
@@ -73,12 +75,17 @@ export class ArgoBot {
     // gets current branch name for pr with a specific number
     private static async getCurrentBranchContext(context, prNumber) {
         // I couldn't find an API call that filters this properly
```

Comment:
I am going to improve this in a future PR, I think there are some performance improvements I can make here.